src/snowflake/cli/plugins/git/manager.py

```diff
@@ -74,6 +75,13 @@ def create(
             query += f"git_credentials = {secret}\n"
         return self._execute_query(query)
 
+    def copy_files(self, source_path: str, destination_path: str) -> SnowflakeCursor:
+        if destination_path.startswith("@~"):
```

Comment:
Let's implement this on `StagePathParts` as `is_user_stage` or somewhere. WDYT?

Reply:
There is separated UserStagePathParts, so I used it.